ui/apps/dashboard/src/app/(organization-active)/(dashboard)/billing/(overview)/page.tsx

```diff
@@ -120,89 +112,67 @@ export default async function Page() {
             entitlement={{
-              currentValue: entitlementUsage.eventSize.limit,
-              displayValue: kbyteDisplayValue(entitlementUsage.eventSize.limit),
-              planLimit: currentPlan.entitlements.eventSize.limit,
+              currentValue: entitlements.eventSize.limit,
+              displayValue: kbyteDisplayValue(entitlements.eventSize.limit),
             }}
-            enableSelfServiceFeatureFlag={enableSelfServiceFF}
           />
           <EntitlementListItem
+            planName={currentPlan.name}
             title="Concurrency"
             description="Maximum number of concurrently executing steps"
             tooltipContent="Functions actively sleeping and waiting for events are not counted"
-            canIncreaseLimitInCurrentPlan={
-              entitlementUsage.isCustomPlan || currentPlan.addons.concurrency.available
-            }
             entitlement={{
-              currentValue: entitlementUsage.concurrency.limit,
-              displayValue: `${entitlementUsage.concurrency.limit} concurrent steps`,
-              planLimit: currentPlan.entitlements.concurrency.limit,
-              maxValue: 1000, // TODO: https://linear.app/inngest/issue/INN-4310/use-maxlimitincurrentplan-data-from-gql-in-the-addons-ui
-            }}
-            addon={{
-              price: currentPlan.addons.concurrency.price,
-              quantityPer: currentPlan.addons.concurrency.quantityPer,
-              addonName: 'concurrency',
+              currentValue: entitlements.concurrency.limit,
+              displayValue: `${entitlements.concurrency.limit} concurrent steps`,
             }}
+            addon={addons.concurrency}
             onChange={refetch}
-            enableSelfServiceFeatureFlag={enableSelfServiceFF}
           />
           <EntitlementListItem
+            planName={currentPlan.name}
             title="Users"
             description="Maximum number of users on the account"
-            canIncreaseLimitInCurrentPlan={
-              entitlementUsage.isCustomPlan || currentPlan.addons.userCount.available
-            }
             entitlement={{
-              currentValue: entitlementUsage.userCount.limit,
-              displayValue: `${entitlementUsage.userCount.usage} of ${entitlementUsage.userCount.limit} maximum users`,
-              planLimit: currentPlan.entitlements.userCount.limit,
-              maxValue: 1000, // TODO: https://linear.app/inngest/issue/INN-4310/use-maxlimitincurrentplan-data-from-gql-in-the-addons-ui
-            }}
-            addon={{
-              quantityPer: currentPlan.addons.userCount.quantityPer,
-              price: currentPlan.addons.userCount.price,
-              addonName: 'user_count',
+              currentValue: entitlements.userCount.limit,
+              displayValue: `${entitlements.userCount.usage} of ${entitlements.userCount.limit} maximum users`,
             }}
+            addon={addons.userCount}
```

Comment:
Just to confirm, in this case the API will return the user addons for Basic and Pro plan, removing the need to verify the plan in the FE right?

Reply:
This object is the `user_count` addon for this specific account, already considering the account's plan